modular_skyrat/modules/clock_cult/code/scriptures/_scripture.dm

```diff
@@ -361,10 +361,8 @@ GLOBAL_LIST_EMPTY(clock_scriptures_by_type)
 	return ..()
 
 
-/datum/action/cooldown/spell/pointed/slab/InterceptClickOn(mob/living/caller, params, atom/clicked_atom)
-	parent_scripture?.click_on(clicked_atom)
-
-
+/datum/action/cooldown/spell/pointed/slab/InterceptClickOn(mob/living/caller, params, atom/target)
+	parent_scripture?.click_on(target)
```

Comment:
This file has two whitespaces between procs (which I think should be how it is everywhere, tbh), removing them would just look weird.
```suggestion
	parent_scripture?.click_on(target)

```